mex.go

```diff
@@ -228,6 +237,12 @@ func (mex *messageExchange) recvPeerFrameOfType(msgType messageType) (*Frame, er
 	}
 }
 
+func (mex *messageExchange) onCtxErr(err error) {
```

Comment:
nit: replace `err` with `_` since we don't do anything with it. maybe worth commenting that the error is reported separately so we don't need to e.g. log it here.